config/networkpolicies/20-ingress-policies.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: csidriver-webhook-ingress
+  namespace: openshift-builds
+spec:
+  podSelector:
+    matchLabels:
+      name: shared-resource-csi-driver-webhook
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: openshift-kube-apiserver
+    ports:
+    - protocol: TCP
+      port: 8443
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: shipwright-webhook-ingress
```

Comment:
The webhook is also deployed by OLM and shouldn't be considered operand. Hence IMO, this should be also be part of the bundle.